wiki/Announcements/Contests/en.md

```diff
@@ -5,7 +5,13 @@
 
 # Contests
 
-## Monthly Beatmapping Contest
+Section dedicated for Contests.
+
+## Official Beatmapping Contests
+
+Official beatmapping contests hosted by the osu!staff.
+
+### Monthly Beatmapping Contest
 
 Notes:
```

Comment:
suggestion:
```
Monthly Beatmapping Contest winners will receive the "Elite Mapper" forum title.
The first to fifth place winners will have their beatmap bundled (along with [osu!tutorial](https://osu.ppy.sh/s/3756)) in the [osu!installer](https://osu.ppy.sh/p/download).
Unless stated otherwise, other runner-up beatmaps are not bundled.
```